aten/src/ATen/native/native_functions.yaml

```diff
@@ -818,10 +818,22 @@
   variants: function, method
 
 - func: fill_(Tensor(a!) self, Scalar value) -> Tensor(a!)
+  cpu_half: True
```

Comment:
can we do this after https://github.com/pytorch/pytorch/pull/20552?  Pretty ugly otherwise...